src/explorer/explorerController.ts

```diff
@@ -38,14 +34,12 @@ export default class ExplorerController {
   };
 
   activateConnectionsTreeView(): void {
-    log.info('Activating explorer controller...');
```

Comment:
Such logs turned out to be distracting and not helpful for debugging, so removing them.